src/components/routes/Account/StripeExpress/StripeConnect.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import AudioLoading from 'shared/loading/AudioLoading';
+
+import { SETTINGS } from 'configs/settings';
+
+const { BEENEST_HOST, STRIPE_CLIENT_ID } = SETTINGS;
+
+const REDIRECT_URL = `${BEENEST_HOST}/account/stripe_express/complete`;
```

Comment:
I still need to test to verify, but it looks like (once the redirect occurs) the two Stripe flows are identical. Do we mind using a `stripe_express` URL in reference to standard Stripe accounts?

Reply:
oh wow, free to to test with existing endpoint.
once we verify, maybe we can rename stripe_express to `stripe_connect` to avoid confusion?